app/src/main/java/com/github/se/cyrcle/ui/authentication/SignIn.kt

```diff
@@ -43,9 +44,12 @@ import kotlinx.coroutines.tasks.await
 fun SignInScreen(navigationActions: NavigationActions, userViewModel: UserViewModel) {
   val context = LocalContext.current
 
+  val failSignInMsg = stringResource(R.string.sign_in_failed_toast)
+  val successSignInMsg = stringResource(R.string.sign_in_successful_toast)
+
   val onAuthComplete = { result: AuthResult ->
     Log.d("Cyrcle", "User signed in: ${result.user?.displayName}")
```

Comment:
I don't know if we should refactor those string what do you think ?

Reply:
As they were display (even in a Toast), I applied the boundary of this PR. Might be a bit overkill, I agree, but as it's already done, I wouldn't bother removing them if it's not blocking for you.

Reply:
Same for me it's not blocking